plugins/MultiSites/API.php

```diff
@@ -384,6 +387,94 @@ function ($value) {
         return $dataTable;
     }
 
+    /**
+     * Temporary/partially mocked data for the all websites dashboard.
+     *
+     * @internal
+     *
+     * @param int $idSite
+     * @param string $period
+     * @param string $date
+     * @param string|false $segment
+     * @param string $pattern
+     * @return array
+     * @throws Exception
+     */
+    public function mockDashboardData(
+        int $idSite,
+        string $period,
+        string $date,
+        int $filter_limit,
+        $segment = false,
+        string $pattern = ''
+    ): array {
+        $featureFlagManager = StaticContainer::get(FeatureFlagManager::class);
+
+        if (!$featureFlagManager->isFeatureActive(ImprovedAllWebsitesDashboard::class)) {
+            throw new Exception('THIS API SHOULD NOT BE USED');
+        }
+
+        Piwik::checkUserHasSomeViewAccess();
+
+        if (Period::isMultiplePeriod($date, $period)) {
+            throw new Exception('Multiple periods are not supported');
+        }
+
+        $numberFormatter = NumberFormatter::getInstance();
+
+        $response = [
+            'totals' => [
+                'hits_evolution'            => $numberFormatter->formatPercent(2.8, 2, 2),
+                'hits_evolution_trend'      => 1,
+                'nb_hits'                   => $numberFormatter->formatNumber(582303),
+                'nb_hits_badge'             => '<strong>Plan:</strong> 600K hits/month',
```

Comment:
Isn't only an information that is relevant to UI only? It's no information that is useful in the API imho. But we can also solve that later, as we need a way to hook into that and change the content anyway.

Reply:
I'll move it to the twig part for now to keep the API cleaner, and it will get sourced from the final place once we know where it could and should belong :+1: